gyp_xwalk

```diff
@@ -339,8 +342,10 @@ if __name__ == '__main__':
   args.extend(['-D', 'gyp_output_dir=' + GetOutputDirectory()])
 
   # Enable Web Audio by default on Android x86
-  if gyp_vars_dict.get('OS') == 'android':
+  if gyp_vars_dict.get('OS') == 'android' and enable_web_audio:
     args.append('-Duse_openmax_dl_fft=1')
```

Comment:
thank you for reminding. yes, disable_webaudio_hrtf also should be set when web audio is disabled.